traceabilityViewer/app/static/components/autocomplete.js

```diff
@@ -208,26 +207,48 @@ app.component("autocomplete", {
         fullInput.value = event.target.value;
         if ( 0 > event.target.selectionStart + correctionNumber ||  event.target.selectionStart + correctionNumber > fullInput.value.length){
           fullInput.value = event.target.value;
-          findSearchValue(event.target.selectionStart)
+          self.pointer = event.target.selectionStart
```

Comment:
`self` in JavaScript? Can't you simply use `pointer`?

Reply:
Of course, resolved in c18ca95e942cb74da2a33660cea9c99a03f51b52